idunn/geocoder/models/geocodejson.py

```diff
@@ -165,6 +174,7 @@ class IntentionDescription(BaseModel):
 
 
 class Intention(BaseModel):
+    type: IntentionType
```

Comment:
There are also many other "type" fields in the API and in the implementation,
but I don't have any better name right now :slightly_smiling_face:

Reply:
We could go for `mode`, `kind`, ... but I can't come up with something that doesn't just look like a synonym :(